src/EFCore.PG/Extensions/NpgsqlFullTextSearchDbFunctionsExtensions.cs

```diff
@@ -77,5 +77,20 @@ public static class NpgsqlFullTextSearchDbFunctionsExtensions
         /// </summary>
         public static NpgsqlTsQuery ToTsQuery(this DbFunctions _, string config, string query) =>
             throw new NotSupportedException();
+
+        /// <summary>
+        /// Convert <paramref name="query" /> tsquery using the simplified websearch syntax.
+        /// http://www.postgresql.org/docs/current/static/textsearch-controls.html#TEXTSEARCH-PARSING-QUERIES
```

Comment:
Thanks for linking to the docs, but can you move them into a separate remarks node?

Reply:
That would be inconsistent with all other functions in the file. Are you sure you want this?